package.json

```diff
@@ -20,13 +20,20 @@
     "jsonwebtoken": "^8.3.0",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
+    "mongoose": "^5.4.21",
     "mongoose-unique-validator": "^2.0.1",
-    "morgan": "^1.9.0",
+    "morgan": "^1.9.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
-    "slug": "^0.9.1",
+    "slug": "^1.1.0",
     "underscore": "^1.9.1"
+  },
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
I seed dependencies but no eslint Config file. You can check if you can push it also

Reply:
Thank you, I pushed it.

Reply:
Yeah, Now It's good!